pkg/device/device-util.go

```diff
@@ -450,6 +497,38 @@ func GetDiskDetails() ([]apis.Device, error) {
 	return result, nil
 }
 
+// ListPartUsed lists all disk partitions created by plugin.
+func ListPartUsed() ([]PartUsed, error) {
```

Comment:
What is the difference between this function and `getAllPartsUsed()`. I can see that they serve similar purpose

Reply:
`getAllPartsUsed` provides all partitions belonging to certain disk whereas `ListPartUsed` inspects all disks and provide their partitions if disk has meta partition. Again, there was no extensibility or straightforward way to change getAllPartsUsed method without doing a major refactoring. As we are planning to move the plugin to lvm localpv soon, let's take up this debt of refactoring at that time.

Reply:
@praveengt Was this involved in your refactoring plan?